test/ssl-test_test.rb

```diff
@@ -6,107 +6,107 @@
     end
 
     it "returns no error on valid SAN" do
       valid, error, cert = SSLTest.test("https://1000-sans.badssl.com/")
-      error.must_be_nil
-      valid.must_equal true
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      _(error).must_be_nil
+      _(valid).must_equal true
+      _(cert).must_be_instance_of OpenSSL::X509::Certificate
     end
 
     it "returns no error when no CN" do
       valid, error, cert = SSLTest.test("https://no-common-name.badssl.com/")
-      error.must_be_nil
-      valid.must_equal true
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      _(error).must_be_nil
+      _(valid).must_equal true
+      _(cert).must_be_instance_of OpenSSL::X509::Certificate
     end
 
     it "works with websites blocking http requests" do
       valid, error, cert = SSLTest.test("https://obyava.ua")
-      error.must_be_nil
-      valid.must_equal true
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      _(error).must_be_nil
+      _(valid).must_equal true
+      _(cert).must_be_instance_of OpenSSL::X509::Certificate
     end
 
     it "returns error on self signed certificate" do
       valid, error, cert = SSLTest.test("https://self-signed.badssl.com/")
-      error.must_equal "error code 18: self signed certificate"
-      valid.must_equal false
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      _(error).must_equal "error code 18: self signed certificate"
+      _(valid).must_equal false
+      _(cert).must_be_instance_of OpenSSL::X509::Certificate
     end
 
     it "returns error on incomplete chain" do
       valid, error, cert = SSLTest.test("https://incomplete-chain.badssl.com/")
-      error.must_equal "error code 20: unable to get local issuer certificate"
-      valid.must_equal false
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      _(error).must_equal "error code 20: unable to get local issuer certificate"
+      _(valid).must_equal false
+      _(cert).must_be_instance_of OpenSSL::X509::Certificate
     end
 
     it "returns error on untrusted root" do
       valid, error, cert = SSLTest.test("https://untrusted-root.badssl.com/")
-      error.must_equal "error code 20: unable to get local issuer certificate"
-      valid.must_equal false
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      _(error).must_equal "error code 19: self signed certificate in certificate chain"
+      _(valid).must_equal false
+      _(cert).must_be_instance_of OpenSSL::X509::Certificate
     end
 
     it "returns error on invalid host" do
       valid, error, cert = SSLTest.test("https://wrong.host.badssl.com/")
-      error.must_equal 'hostname "wrong.host.badssl.com" does not match the server certificate'
-      valid.must_equal false
-      cert.must_be_instance_of OpenSSL::X509::Certificate
+      assert error.include?('hostname "wrong.host.badssl.com" does not match the server certificate')
```

Comment:
I have the version before you changed it on my machine so I changed the matcher so work with both